### DISCUSSION TITLE
Connect Custom Domain Verification

### DISCUSSION THREAD

**Discussion by dannyleungkc** [swellstores/community#40] [Help] [answered]
Hi there,

I have an issue with Custom Domain Verification. I'm using Cloudflare as Swell suggested to manage my custom domains. I set it up and It works fine with my custom domain. However, I notice when I go to the checkout page of my store. The url becomes Swell's default one. **detoorp.swell.store/checkout/** instead of having my custom domain at the front. I looked at help centre and say I have to verify my domain at the Storefront>Domain in the admin page.

However, since I'm using Cloudflare, I'm unable to verify it. Even though the set-up is already completed and my store is already connected to my custom domain www.detoorp.com

Could you please help look at it?

<img width="389" alt="Screenshot 2022-12-18 at 23 57 20" src="https://user-images.githubusercontent.com/119118081/208326332-c1c7a55b-aec7-4e13-8029-a001c4bfded3.png">

Regards,
D

**Comment by ericingram** [comment]
Someone will help you figure that out, but I wanted to say your site looks really nice.

**Reply by dannyleungkc** [reply]
> Someone will help you figure that out, but I wanted to say your site looks really nice.

Oh thank you Eric. I'm only using Origin template. But I'm happy with what I've built from it so far :^)
Let me know if you wanna feature my site in one of your marketing campaigns at some point LOL 😜😜😜

**Reply by ericingram** [reply]
Yes! We're working on a way to share more of what developers are building, more frequently. cc @ana-estrela

**Reply by ana-estrela** [reply]
@dannyleungkc 😮 This is looking so good! Once everything is set up and running, I encourage you to share it in our [show and tell](https://github.com/orgs/swellstores/discussions/categories/show-and-tell) and talk about what were some of the interesting things you discovered along the way! 🙌 
 
As eric mentioned, we want to share more of the work developers like yourself have been working on! More to come real soon. 😄

**Comment by dannyleungkc** [comment]
I wonder if anyone can help with this? :)

**Reply by greph** [reply]
Hi @dannyleungkc , in regards to the checkout, there's a setting within Storefront > Settings which allows you to specify a custom checkout url instead of the Swell-hosted one. Just remember to include the {checkout_id} placeholder, like so: https://mystore.com/checkout/{checkout_id}.

**Comment by logeshswell** [comment, marked as answer]
@dannyleungkc I just tried DNS lookup for your domain and found 2 things that you missed:
1. `A` records should be removed.
2. `CNAME` record should be created.

Please follow the steps described in this documentation:
https://developers.swell.is/storefronts/custom-domains-and-ssl-with-cloudflare#editing-your-dns-records-on-cloudflare

Once you completed the above 2 actions, then you can verify the domain in the Swell dashboard after a few minutes.

**Screenshots for reference:**
<img width="512" alt="Screenshot 2023-01-12 at 1 28 37 AM" src="https://user-images.githubusercontent.com/100766131/211905522-2989b7b4-738c-48a8-ad64-1800a57c64ea.png">

<img width="917" alt="Screenshot 2023-01-12 at 1 14 33 AM" src="https://user-images.githubusercontent.com/100766131/211905278-832742b3-426f-4905-ae35-681483174dc4.png">
